core/src/test/java/smile/regression/GradientTreeBoostTest.java

```diff
@@ -64,6 +66,9 @@ public void testLongley() throws Exception {
         MathEx.setSeed(19650218); // to get repeatable results.
         GradientTreeBoost model = GradientTreeBoost.fit(Longley.formula, Longley.data);
 
+        TreeShapExplainer shapExplainer = new TreeShapExplainer(model.trees(), true);        
```

Comment:
This is NOT real test. Can you get the result from other implementation and compare to them here?

Reply:
Thanks for review. will get implementation comparison here